map_org_names.yaml

```diff
@@ -273,8 +273,8 @@ mappings:
   - ['^[[:space:]]*astek[[:space:]]*$','Astek Corporation']
   - ['^[[:space:]]*astrafox[[:space:]]*$','ASTRAFOX']
   - ['^[[:space:]]*astrazeneca[[:space:]]*$','AstraZeneca Pharmaceuticals']
-  - ['^[[:space:]]*cingular[[:space:]]*$','AT&T Inc']
-  - ['^[[:space:]]*at&?t[[:space:]]*$','AT&T Services, Inc.']
+  - ['^[[:space:]]*cingular[[:space:]]*$','AT&T Services, Inc.']
+  - ['^[[:space:]]*a[[:space:]]*t[[:space:]]*&[[:space:]]*t[[:space:]]*(telecommunication|labs|entertainment)?[[:space:]]*,?[[:space:]]*(inc|group)?.?$','AT&T Services, Inc.']
```

Comment:
actually `.?` is incorrect, you should escape dot `\.?` - optional dot.

Reply:
thank you! I just missed that, the query worked well with SQL regex. Updated.